### ISSUE TITLE
[FEATURE] Add some questions related to bit manipulation

### ISSUE THREAD

**Issue by literalEval** [TheAlgorithms/C-Plus-Plus#2052]
### Detailed description

Some more questions can be added on this topic like "Given an array of numbers that repeat twice, except a single number that occurs only once. Find the single occurring number" and some others.

### Context

There are very few questions on this topic, and adding few more would be good for practice.

**Comment by kgpjay** [comment]
Hi can I take on this issue?

**Comment by literalEval** [comment]
I have already done this in #2061 
Maintainers please have a look at it.

**Comment by bagariaraj23** [comment]
hi, can i contribute with some questions on bit-manipulation ?

**Comment by SHWETA2JHA** [comment]
Please assign this task to me.

**Comment by BugsBunny-PG** [comment]
Hello Maintainer,
I want to solve this problem ,please assign this issue to me.

**Comment by Sakshi2002-Sinha** [comment]
Hello, Can you please assign this issue to me?
Thanks in advance :)
